src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/RequestMappingService.java

```diff
@@ -160,9 +161,23 @@
         for (QueryResultsRow row : queryResults) {
             roleAssignments.add((RoleAssignment) row.get("$roleAssignment"));
         }
+        logRejectedAccessProfiles(allProfiles, roleAssignments);
         return roleAssignments;
     }
 
+    public static void logRejectedAccessProfiles(Set<UserAccessProfile> userAccessProfiles,
+                                                                    List<RoleAssignment> roleAssignments) {
+        List<UserAccessProfile> userAccessProfileList = (userAccessProfiles.stream()
+                .filter(two -> roleAssignments.stream()
```

Comment:
variable name should be proper instead of one and two.

Reply:
We have already written the logic to identify the invalid profile at line no 98. Hence, we don't need this method separately. you can just add the logger to print the invalid profile.